### DISCUSSION TITLE
gdsfactory: gf tool install failed

### DISCUSSION THREAD

**Discussion by ghost** [gdsfactory/gdsfactory#534] [Q&A] [answered]
Hi all
I am new to gdsfactory.
I tried to run the command: gf tool install for Klayout integration.
However, there is an error: [WinError 3] The system cannot find the path specified: 'C:\\Users\\s3923133\\KLayout\\tech\\generic'
I tried both ways illustrated on: https://gdsfactory.github.io/gdsfactory/notebooks/_2_klayout.html
Could you please help me ?

**Comment by thomasdorch** [comment]
Hi, welcome! 

Does the 'C:\Users\s3923133\KLayout\tech' directory exist? If not, try creating it and running the command again. If that doesn't work can you provide the full error message it prints in the console?

**Reply by ghost** [reply]
Thank you so much ! That's work.

**Comment by joamatab** [comment, marked as answer]
Thank you Thomas and @hiepdxphysics

just released version 5.12.17 with a fix for the issue

https://github.com/gdsfactory/gdsfactory/commit/b84f29f5a778fba83987ab80f218b0623c47acd7

let me know how that works
